http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcConfigurationServletListener.java

```diff
@@ -66,7 +67,7 @@ public void contextInitialized(ServletContextEvent sce) {
                 if (is == null) {
                     oidcClientConfiguration = new OidcClientConfiguration();
                 } else {
-                    oidcClientConfiguration = OidcClientConfigurationBuilder.build(is);
+                    oidcClientConfiguration = OidcClientConfigurationBuilder.buildWithoutUnsupportedAttributes(is, servletContext.getInitParameter(JSON_CONFIG_UNSUPPORTED_ATTRIBUTE_PARAM));
```

Comment:
Same comment as on the other PR:

https://github.com/wildfly-security/wildfly-elytron/pull/1925/files#r1532368580